.babelrc

```
@@ -1,4 +1,4 @@
 {
-  "presets": [["@babel/preset-env", { "targets": { "node": "6" }, "useBuiltIns": "usage" }]],
+  "presets": [["@babel/preset-env", { "targets": { "node": "14" }, "useBuiltIns": "usage" }]],
```

This is a major semver change in the JS emit, and it's not worth it for that

```suggestion
  "presets": [["@babel/preset-env", { "targets": { "node": "6" }, "useBuiltIns": "usage" }]],
```